Jenkinsfile

```diff
@@ -27,6 +20,11 @@ node('ubuntu-zion') {
   GitHub gitHub
 
   try {
+    stage('Init IQ Version & Sha') {
+      nexusIqVersion = getVersionFromBuildName(env.releaseBuild_NAME)
+      nexusIqSha = readBuildArtifact('insight/insight-brain/release', env.releaseBuild_NUMBER,
```

Comment:
The `readBuildArtifact` method is being added in https://github.com/sonatype/jenkins-shared/pull/140